features/all-positions/PositionActionsDialog.tsx

```diff
@@ -489,19 +504,69 @@ const PositionActionsDialog = (props: DialogProps) => {
                           price.
                           <br></br>
                           <br></br>
+                          <Status>
+                            <Label>Current collateral/token ratio: </Label>
```

Comment:
This table is super useful. However, I think it's pretty complex unless you understand the details of our liquidation mechanism. Could we add some hover information explaining in concise terms that explains what each of these values are? 

The same could be said for the 4 input properties below. For those we could even take the contract natspec directly from the solidity files.

Reply:
I added tool tip above the table:
![Screen Shot 2020-08-27 at 08 27 22](https://user-images.githubusercontent.com/9457025/91441994-2c2dbc80-e83f-11ea-89ff-eb875cfa1407.png)